docs/sources/setup/install/helm/reference.md

```diff
@@ -2896,7 +2896,7 @@ null
 			<td><pre lang="json">
 {
   "name": "self-monitoring",
-  "secretNamespace": "{{ .Release.Namespace }}"
+  "secretNamespace": "{{ include "loki.namespace" . }}"
```

Comment:
Did you edit this file directly? Or was this change automatically generated? The Helm reference should be updated in `production/helm/loki/reference.md.gotmlp`. There's a note to that effect at line 11 of the `docs/sources/setup/install/helm/reference.md` file, you might have missed it.

Reply:
Yes, I missed it. Thanks for the heads up :)